docs/localized/KR/README.md

```diff
@@ -54,7 +54,7 @@ ML-Agents toolkit은 유니티의 풍부한 환경에서 인공지능 에이전
 ## 커뮤니티 그리고 피드백
 
 ML-Agents toolkit은 오픈소스 프로젝트이며 컨트리뷰션을 환영합니다. 만약 컨트리뷰션을 원하시는 경우
-[컨트리뷰션 가이드라인](CONTRIBUTING.md)과 [행동 규칙](CODE_OF_CONDUCT.md)을 검토해주십시오.
```

Comment:
The localized documentation is frozen and should not change. It is tied to a specific version.

Reply:
That would make sense if the links were tied to that specific version, but they are not.  Do you want me to revert those changes?